src/tools/compiletest/src/main.rs

```diff
@@ -320,6 +326,10 @@ pub fn parse_config(args: Vec<String>) -> Config {
         run_ignored,
         filter: matches.free.first().cloned(),
         filter_exact: matches.opt_present("exact"),
+        force_pass_mode: matches.opt_str("pass").map(|mode|
+            mode.parse::<PassMode>()
+                .unwrap_or_else(|_| panic!("unknown `--pass` option `{}` given.", mode))
```

Comment:
```suggestion
                .unwrap_or_else(|_| panic!("unknown `--pass` option `{}` given", mode))
```

Reply:
(Diagnostics like this are not "full sentences" AFAIK, so they start with a lower case letter and don't end with a dot.)